renku/core/commands/doctor.py

```diff
@@ -18,6 +18,7 @@
 """Check your system and repository for potential problems."""
 import traceback
 
+from renku.cli.utils.callback import ClickCallback
```

Comment:
communicator should be set in `renku.cli` (before calling .build()), `renku.core` shouldn't have a dependency on `renku.cli`